### PR TITLE
[54288] wording is off for progress calculation input errors

### PR DESCRIPTION
**Pull request by bsatarnejad**
https://community.openproject.org/projects/openproject/work_packages/54288/activity

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1065,12 +1065,14 @@ Project attributes and sections are defined in the <a href=%{admin_settings_url}
               only_same_project_categories_allowed: "The category of a work package must be within the same project as the work package."
               does_not_exist: "The specified category does not exist."
             estimated_hours:
-              cant_be_inferior_to_remaining_work: "can't be inferior to 'Remaining work'."
+              cant_be_inferior_to_remaining_work: "Cannot be lower than Remaining Work."
               must_be_set_when_remaining_work_is_set: "must be set when 'Remaining work' is set."
```
Comment (psatyal) [inline config/locales/en.yml]: I would make this one: "Required when Remaining work is set."

File: config/locales/en.yml [hunk 2]
```diff
@@ -1065,12 +1065,14 @@ Project attributes and sections are defined in the <a href=%{admin_settings_url}
...
               only_values_greater_or_equal_zeroes_allowed: "must be >= 0."
+              format: "%{message}"
             remaining_hours:
-              cant_exceed_work: "can't exceed 'Work'."
+              cant_exceed_work: "Cannot be higher than work."
               must_be_set_when_work_is_set: "must be set when 'Work' is set."
```
Comment (psatyal) [inline config/locales/en.yml]: And this one: "Required when Work is set."

**Review by psatyal — CHANGES_REQUESTED** [review]
Hi @bsatarnejad. I've included suggestions for the two error messages I hadn't initially covered. No quotation marks necessary and we'll keep the capitalisation like this ("Remaining work" or "Work", with only the first letter being capitalised).

**Review by HDinger — CHANGES_REQUESTED** [review]
Please review the usages of `Remain work` and `Work` and take care that the first word is always capitalized. Currently, there is a wild mixture of capitalized and lower case words.

File: config/locales/en.yml [hunk 3]
```diff
@@ -1065,12 +1065,14 @@ Project attributes and sections are defined in the <a href=%{admin_settings_url}
...
+              cant_be_inferior_to_remaining_work: "Cannot be lower than Remaining Work."
+              must_be_set_when_remaining_work_is_set: "Required when Remaining work is set."
+              only_values_greater_or_equal_zeroes_allowed: "Must be >= 0."
+              format: "%{message}"
             remaining_hours:
-              cant_exceed_work: "can't exceed 'Work'."
-              must_be_set_when_work_is_set: "must be set when 'Work' is set."
+              cant_exceed_work: "Cannot be higher than work."
+              must_be_set_when_work_is_set: "Required when Work is set."
+              format: "%{message}"
```
Comment (HDinger) [inline config/locales/en.yml:1075]: What are these strings needed for?

Comment (bsatarnejad) [reply config/locales/en.yml:1075]: Hi @HDinger 
Sorry, I just copied the text from WP description, I will correct them.

According to [this blog post](https://www.bigbinary.com/blog/rails-6-allows-to-override-the-activemodel-errors-full_message-format-at-the-model-level-and-at-the-attribute-level?ref=writesoftwarewell.com), in rails the default format for showing validation error messages related to models' attributes is `%{attribute} %{message}`, but We can override the format of error messages by changing `errors.format`.

In our case, we don't want to show the attribute name at the beginning of the error sentence, however, show just the error message related to that attribute.

**Review by HDinger — APPROVED** [review]

**Pull request merged**